sb3_contrib/common/policies.py

```diff
@@ -0,0 +1,28 @@
+"""Policies: abstract base class and concrete implementations."""
+
+from stable_baselines3.common.distributions import Distribution
+from stable_baselines3.common.policies import ActorCriticPolicy as _ActorCriticPolicy
+
+
+class ActorCriticPolicy(_ActorCriticPolicy):
+    """
+    Policy class for actor-critic algorithms (has both policy and value prediction).
+    Used by A2C, PPO and the likes.
+    """
+
+    def get_distribution(self) -> Distribution:
```

Comment:
why do you need an additional method?
For clarity?
In that case, best is to probably add that method in SB3 directly, no? (+ make it compute the distribution given observation)

Reply:
I followed what was discussed here: https://github.com/Stable-Baselines-Team/stable-baselines3-contrib/issues/38#issuecomment-898883577
I only added it in SB3-contrib to avoid having discussions scattered across multiple repo but ultimately everything will end up in SB3.
I'll open a PR in SB3.

Reply:
Opened the PR: https://github.com/DLR-RM/stable-baselines3/pull/564